### PR TITLE
Datatable

### PR DESCRIPTION
**Pull request by joefitter**
A sortable datatable react component. Also added front end architecture and example

### CODE REVIEW

File: example/schema/places.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  holding: {
+    show: true,
+    sortable: false
+  },
+  nacwo: {
+    show: true,
+    accessor: 'nacwo.profile.name'
+  },
+  updatedAt: {}
+};
```
Comment (lennym) [inline example/schema/places.js]: Can you make this file a bit less ASL specific please?

File: example/views/index.jsx [hunk 2]
```diff
@@ -8,8 +9,19 @@ import {SubNavBar, NavItem} from '../../components/subnavbar';
...
+import { TabBar } from '../../components/tabbar';
+import { TabItem } from '../../components/tabbar/tabitem';
+import DataTable from '../../components/datatable';
+import places from '../data/places';
+import schema from '../schema/places';
+
+const formatters = {
+  suitability: { format: arr => arr.join(', ') },
+  holding: { format: arr => arr.join(', ') },
+  nacwo: { title: key => key.toUpperCase() }
```
Comment (lennym) [inline example/views/index.jsx]: Again, can we come up with a _super_ generic example dataset?

File: package.json [hunk 3]
```diff
@@ -12,7 +13,9 @@
     "test:watch": "npm run test:unit-source -- --watch",
     "test:update": "npm run test:unit-source -- --updateSnapshot",
     "build": "npm run build:js && npm run build:css",
+    "build:watch": "npm run build:js -- --watch | npm run build:css -- --watch",
```
Comment (lennym) [inline package.json]: Does the `--watch` flag work on the css build? I'll be damned!

Comment (joefitter) [reply package.json]: nope, forgot to remove that 😀

File: example/views/index.jsx (hunk 2 above)
Comment (joefitter) [reply example/views/index.jsx]: yeppo

File: example/schema/places.js (hunk 1 above)
Comment (joefitter) [reply example/schema/places.js]: yeppo

**Pull request closed without merging**